packages/nimble-components/src/dialog/tests/dialog.stories.ts

```diff
@@ -154,6 +192,20 @@ const metadata: Meta<DialogArgs> = {
                 }
             }
         },
+        size: {
+            description: sizeDescription,
+            options: [
+                DialogSizeOptions.smallGrowable,
+                DialogSizeOptions.largeFixed
+            ],
+            control: {
+                type: 'select',
```

Comment:
can you make this a radio instead? makes it easier to see the available options and the list is small so we can show them all

Reply:
changed 
![image](https://github.com/ni/nimble/assets/57374665/881570b6-7e4e-4682-9ef4-d5792504a727)